recipes/openexr/2.x/conanfile.py

```diff
@@ -14,11 +14,11 @@ class OpenEXRConan(ConanFile):
     name = "openexr"
     description = "OpenEXR is a high dynamic-range (HDR) image file format developed by Industrial Light & " \
                   "Magic for use in computer imaging applications."
-    topics = ("openexr", "hdr", "image", "picture")
+    topics = ("hdr", "image", "picture", "file format", "computer vision")
```

Comment:
Should topics use `-` instead of spaces?

Reply:
Thats a bug in the webui, I do not think it's important for the actual recipes

Reply:
yes, the webui was broken before with topics, but doing some effort over the search engine we can merge those topics based on words.